### DISCUSSION TITLE
Installazione

### DISCUSSION THREAD

**Discussion by AlessandroFalivene** [Home-Assistant-Pro-Team/Gestione-Allarme-Home-Assistant-Alarm#6] [Q&A]
Ciao. Vorrei provare il pacchetto che avete sviluppato che mi sembra molto completo.
Mi trovo però bloccato subito durante l'installazione e configurazione. 

In particolare questo punto:

`homeassistant:`
`  customize:`
`    package.node_anchors:`
`        Allarme: &alarm alarm_control_panel.home_alarm`

ottengo errore di configurazione

Error loading /config/configuration.yaml: mapping values are not allowed here
in "/config/configuration.yaml", line 6, column 25

questo è il mio yaml:

`homeassistant:`
`  packages: !include_dir_named packages`
`  media_dirs:`
`    media: /media`
`  customize: !include customization.yaml`
`    package.node_anchors:`
`        Allarme: &alarm alarm_control_panel.home_alarm`

Grazie per il supporto e complimenti per il lavoro fatto!

**Comment by AlessandroFalivene** [comment]
Come non detto....dovevo metterli nel file customization.yaml.... 

Ora però controllando la validità dello yaml ottengo questi errori:

Error loading /config/configuration.yaml: while constructing a mapping
in "/config/packages/alarm/skill_device/control_code/tastierino estero/ESPHOME_tastierino.yaml", line 1, column 1
expected a mapping or list of mappings for merging, but found scalar
in "/config/packages/alarm/skill_device/control_code/tastierino estero/ESPHOME_tastierino.yaml", line 13, column 5

Suggerimenti?

**Comment by CasaMarcelli** [comment]
Ciao. Sto provando Homeassistant e ho visto il tuo pacchetto molto interessante e vorrei provarlo. Come posso fare per scaricarlo? C'è un link diretto? Hai previsto anche una reposity da richiamare fra i componenti aggiuntivi richiamabili esternamente (archivi digitali)?

**Reply by marco-hacs** [reply]
Questo non custom componente ma un packages, devi seguire le istruzioni del readmi.

**Reply by CasaMarcelli** [reply]
Grazie della veloce risposta. Ho letto il readme  ma quello che non riesco a capire è come fare il download dei files e cartelle. Esiste un link diretto o uno zip? Di solito su questo sito trovo dei link per lo scarico...

**Reply by marco-hacs** [reply]
https://github.com/Home-Assistant-Pro-Team/Gestione-Allarme-Home-Assistant-Alarm 
![image](https://github.com/Home-Assistant-Pro-Team/Gestione-Allarme-Home-Assistant-Alarm/assets/62516592/f9fe93ac-fc2f-4623-a508-daaca2e842dd)

**Reply by CasaMarcelli** [reply]
Grazie...ho capito perché non vedevo il link di scarico. Stavo leggendo la pagina da smartphone e solo visualizzando in modalità desktop si riesce a vedere il link che mi hai indicato. Adesso provo il tutto...ancora grazie.
